cogs/misc.py

```diff
@@ -44,6 +86,36 @@ async def _8ball(self, ctx, *, question=None):
             random.choices(tuple(outcomes.keys()), tuple(outcomes.values()))[0]
         )
 
+    @commands.command(name="proton", aliases=["protondb"])
+    async def proton(self, ctx):
+        """Retrieve ProtonDB stats for Project Winter."""
+
+        delta_time = time.time() - self.proton_pw["time"]
+        delta_time_str = datetime.fromtimestamp(delta_time).strftime('%M')
+
+        if delta_time_str[0] == "0":
+            delta_time_str = delta_time_str[0]
+
+        # Construct an embed with queried data
+
+        embed = Embed(
+            title="Project Winter Proton Statistics",
+            colour=self.proton_pw["colour"]
+        )
+
+        for key, value in self.proton_pw["data"].items():
+            embed.add_field(name=key, value=value, inline=False)
+
+        match delta_time_str:
+            case "0":
+                embed.set_footer(text="Last updated now.")
+            case "1":
+                embed.set_footer(text="Last updated 1 minute ago.")
+            case _:
+                embed.set_footer(text=f"Last updated {delta_time_str} minutes ago.")
```

Comment:
We don't use 3.10 on the bot yet, so this will have to be reverted to if-statements.

Reply:
I'll have to update the poetry config if that's the case

Reply:
I did so before I merged it